lib/iris/tests/test_cube.py

```diff
@@ -88,5 +88,14 @@ class Terry:
         self.assertIs(cube.__ne__(Terry()), NotImplemented)
 
 
+class Test_CubeList_copy(tests.IrisTest):
```

Comment:
This belongs in [`tests/unit/cube/test_CubeList.py`](https://github.com/SciTools/iris/blob/master/lib/iris/tests/unit/cube/test_CubeList.py)

I fully acknowledge Iris' testing structure is unclear!